src/core/component.js

```diff
@@ -50,10 +50,22 @@ var Component = module.exports.Component = function (el, attrValue, id) {
 
   // Store component data from previous update call.
   this.attrValue = undefined;
-  this.nextData = this.isObjectBased ? this.objectPool.use() : undefined;
```

Comment:
Perhaps I'd just add a `keysFilter` argument to `objectPool.use` and pass that in versus a separate function.

Reply:
@3DataWill What do you think about that?